### PR TITLE
Polish translation improvement in Validator component

### PR DESCRIPTION
**Pull request by szepczynski**
| Q             | A
| ------------- | ---
| Branch?       | 2.7 
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | no
| License       | MIT

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf [hunk 1]
```diff
@@ -36,11 +36,11 @@
             </trans-unit>
             <trans-unit id="9">
                 <source>This field was not expected.</source>
-                <target>To pole nie spodziewano.</target>
+                <target>Tego pola się nie spodziewano.</target>
             </trans-unit>
             <trans-unit id="10">
                 <source>This field is missing.</source>
-                <target>To pole jest chybianie.</target>
+                <target>To pole jest brakujące.</target>
```
Comment (jakzal) [inline src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf]: Perhaps "Tego pola brakuje." read better?

Comment (szepczynski) [inline src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf]: definitely!

**Comment by jakzal** [comment]
:+1:

File: src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf [hunk 2]
```diff
@@ -36,11 +36,11 @@
             </trans-unit>
             <trans-unit id="9">
                 <source>This field was not expected.</source>
-                <target>To pole nie spodziewano.</target>
+                <target>Tego pola się nie spodziewano.</target>
```
Comment (szepczynski) [inline src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf]: @daryush "To pole nie było spodziewane." doesn't sound like polish

**Comment by fabpot** [comment]
Thank you @szepczynski.

**Pull request closed without merging**